adr/2022-08-decouple-from-ietf.md

```diff
@@ -0,0 +1,137 @@
+more stable place in the future. The main concern is the pain this process is
+causing while we are in this unusual phase of simultaneous unstable growth and
+production use. Standardization isn't out of the question, it's just not
+productive for us to be developing JSON Schema within the constraints of a
+standards organizations procedures.
+
+Option 1 was rejected because it ignores the problems we've been facing and
+provides no solution. No one wants this.
+
+Option 2 was rejected for several reasons. If we go all in with IETF, we would
+have to join a working group and treat JSON Schema like a normal I-D. That means
+we would have to start treating drafts as drafts, which means not recommending
+production use until we are ready for RFC and not releasing a new
+production-ready version of JSON Schema until we've reached RFC status. Most of
+the core contributors don't believe that we are close enough to an RFC-ready
+release that we want to commit to not being able to issue another release until
+that happens.
+
+There are other concerns including skepticism that even with an extension
+mechanism that the RFC wouldn't need regular updates, which is not normal
+practice for an RFC and would require significant effort to issue a replacing
+RFC. Without a concrete proposal on the scope of the RFC and the extension
+mechanisms, it's hard to commit to this path.
+
+Additionally, many of the core contributors have found working with the IETF
+unproductive and have concerns about JSON Schema getting deeper involved without
+compelling enough reason. Most agree that the reasons are not sufficiently
+compelling at this point.
+
+Option 3 was rejected because it has the same problems as Option 2 accept that
+we don't have the same unpleasant history with W3C than we do with IETF.
+
+### Positive Consequences
+
+* Decoupling from IETF allows us to distance ourselves from the assumptions that
+  people make about JSON Schema because they assume it works like a typical I-D.
+* Decoupling from IETF allows us to customize our SDLC model to what works best
+  for JSON Schema.
+
+### Negative Consequences
+
+* Not being associated with a recognized standards organization such as IETF,
+  W3C, or IEEE reduces the credibility of JSON Schema in the eyes of some.
+  However, we have received feedback from people involved in standards
+  development that say they are comfortable adopting OpenAPI without it being
+  associated with a standards organization, so we don't expect this to be a
+  significant issue for JSON Schema either.
+* If we don't go the standardization route with IETF or W3C, we lose access to
+  their expert review process.
+* One of the benefits of an RFC is other standards can normatively reference it,
+  and use JSON Schema to define their JSON-based syntaxes. Independently
+  publishing a specification does not permit this.
+* Defining our own SLDC process will be a lot of work and none of us have
```

Comment:
🙃

```suggestion
* Defining our own SLDC process will be a lot of work and most of us have
```

Reply:
Can you elaborate. We all have experience with what we've been doing with JSON Schema and IETF. Some of us are familiar with OpenAPI's process. Some of us have looked into what other specifications are doing. But, I wouldn't consider any of us experts beyond what we do and what IETF does. I think we have enough experience and are informed enough to take a stab at defining our own process, but we're not experts.

In case it was confusing, the "S" in SLDC stands for "specification", not "software". If it was "software", I would say we have the necessary expertise, but "specification" is a different animal.

Reply:
SpLDC? 🙃